static/js/file-validation.js

```diff
@@ -6,10 +6,13 @@
   if (resumeFileInput) {
     resumeFileInput.addEventListener("change", function () {
       const filePath = resumeFileInput.value;
+      const fileSize = resumeFileInput.files[0].size;
       if (!allowedExtensions.exec(filePath)) {
-        alert(
-          "Invalid file format selected. Allowed formats are: pdf, doc, docx, txt, rtf"
-        );
+        alert("Invalid file format selected. Allowed formats are: pdf, doc, docx, txt, rtf");
+        resumeFileInput.value = "";
+      }
+      if (fileSize >= maxAllowedSize) {
```

Comment:
I couldn't QA this, `maxAllowedSize` seems not to be defined anywhere, and I tested using a 4.2mb PDF, and it somehow submitted successfully:
![Screenshot from 2021-11-29 16-18-15](https://user-images.githubusercontent.com/2376968/143903907-32139dc4-c05f-424f-ad5b-aec005023314.png)